docs/src/components/MainSectionCard.js

```diff
@@ -0,0 +1,97 @@
+// @flow strict
+import { Box, Text } from 'gestalt';
+import * as gestalt from 'gestalt'; // eslint-disable-line import/no-namespace
+import DatePicker from 'gestalt-datepicker';
+import React, { type Node } from 'react';
+import { LiveProvider, LiveError, LivePreview } from 'react-live';
+import ExampleCode from './ExampleCode.js';
+import theme from './atomDark.js';
+import Markdown from './Markdown.js';
+
+type Props = {|
+  defaultCode: string,
```

Comment:
nit: unless there's another obvious ordering scheme (like with the card size name map), I'm a big fan of alphabetizing lists of props. we could also add a lint rule for this (we tried that before in Pinboard, but it was…controversial 😂 )

Reply:
In Pinboard maybe it was controversial, it'd be great for Gestalt.